src/app/[locale]/dataset-list.tsx

```diff
@@ -6,6 +6,23 @@ import {SearchResult} from '@/lib/dataset-fetcher';
 import DatasetCard from './dataset-card';
 import FilterSet from './filter-set';
 import {clientSearchDatasets} from './client-search-dataset';
+import Paginator from './paginator';
+import {
+  PageSidebar,
+  PageContent,
+  PageTitle,
+  PageHeader,
+} from '@/components/page';
+
+export enum SortBy {
```

Comment:
Just a thought: would it be an idea to tailor the `SortBy` and `SortOrder` enums to the need of the UI? Now they're derived from the dataset fetcher's enum. For example, the UI presents three options:

relevance_asc
name_asc
name_desc

We could create just one enum that combines these, and have the `/api/datasets.ts` parse the enum's values into separate `sortBy` and `sortOrder` values, before feeding these to the dataset fetcher?

Reply:
Yes, sounds like an excellent idea